specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-10-01/mfe.json

```diff
@@ -9547,6 +9556,23 @@
       "x-ms-discriminator-value": "AzureDataLakeGen2",
       "additionalProperties": false
     },
+    "AzureDatastore": {
+      "description": "Base definition for Azure datastore contents configuration.",
+      "type": "object",
+      "properties": {
+        "resourceGroup": {
+          "description": "Azure Resource Group name",
+          "type": "string",
+          "x-nullable": true
```

Comment:
>  "x-nullable": true [](http://example.com/codeflow?start=8&length=20)

I think its probably preferred not to use x-nullable, but just to allow omitting them wen unknown.

Reply:
I thought "x-nullable" just makes the "null" as acceptable value? How to mark omit null value in swagger?

Reply:
We prefer implementations to *not* rely on the value null if they don't have to.